features/GEM111_No_duplicated_points_within_a_polyloop_or_polyline.feature

```diff
@@ -0,0 +1,23 @@
+@implementer-agreement
+@GEM
+Feature: GEM111 - No duplicated points within a polyloop or polyline
+
+  Scenario: Agreement on no duplicated points within a polyloop
+
+      Given A file with Schema Identifier "IFC2X3"
+      And An IfcPolyLoop
+      Then The IfcPolyLoop must have no duplicate points
+
+  Scenario: Agreement on no duplicated points within a polyline
+
+      Given A file with Schema Identifier "IFC2X3"
+      And An IfcPolyLine
+      And IfcPolyLine first and last point are different
```

Comment:
We have something similar here (still not merged) with "... forms an open/closed curve". https://github.com/buildingSMART/ifc-gherkin-rules/pull/3/files#diff-e883e365368e1e08c63684a4c4978f76f354dad388f6d71c41e45051063046efR8

I think we should try and unify this.

Reply:
Cool. I've made an attempt to unify this, took most of your code and tweaked it just a little bit, to make it fit my case.